hydra-node/src/Hydra/Network/Heartbeat.hs

```diff
@@ -110,8 +110,10 @@ updateStateFromOutgoingMessages ::
   TVar m HeartbeatState ->
   Network m (Heartbeat (Message msg)) ->
   Network m (Message msg)
-updateStateFromOutgoingMessages localhost heartbeatState Network{broadcast} =
-  Network $ \msg -> do
+updateStateFromOutgoingMessages localhost heartbeatState Network{broadcast, getPeers, setPeers} =
+  Network bcast getPeers setPeers
+ where
+  bcast = \msg -> do
     now <- getMonotonicTime
     updateLastSent heartbeatState now
     broadcast (Data localhost msg)
```

Comment:
Same remark as above here, avoid introducing needless let-bindings (`getPeers`, `setPeers`) in scope.